src/main/java/uk/gov/hmcts/darts/courthouse/controller/CourthousesController.java

```diff
@@ -0,0 +1,208 @@
+package uk.gov.hmcts.darts.courthouse.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.enums.ParameterIn;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import jakarta.persistence.EntityNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.darts.courthouse.api.CourthousesApi;
+import uk.gov.hmcts.darts.courthouse.mapper.CourthouseToCourthouseEntityMapper;
+import uk.gov.hmcts.darts.courthouse.model.Courthouse;
+import uk.gov.hmcts.darts.courthouse.model.ExtendedCourthouse;
+import uk.gov.hmcts.darts.courthouse.service.CourthouseService;
+
+import java.util.List;
+import javax.validation.Valid;
+
+@RestController
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+public class CourthousesController implements CourthousesApi {
+
+    @Autowired
+    CourthouseService courthouseService;
+
+    @Autowired
+    CourthouseToCourthouseEntityMapper mapper;
+
+    /**
+     * DELETE /courthouses/{courthouse_id} : Deletes the courthouse entry with the supplied id.
```

Comment:
Remove all of these comments, not needed

Reply:
ACCEPT